internal/resourcemodifiers/resource_modifiers.go

```diff
@@ -117,7 +118,24 @@ func (r *ResourceModifierRule) PatchArrayToByteArray() ([]byte, error) {
 }
 
 func (p *JSONPatch) ToString() string {
-	if strings.Contains(p.Value, "\"") {
+	// if value is empty, then add empty quotes
+	if p.Value == "" {
+		return fmt.Sprintf(`{"op": "%s", "from": "%s", "path": "%s", "value": ""}`, p.Operation, p.From, p.Path)
+	}
+	// if value is null, then don't add quotes
```

Comment:
what about creating a function like this
```go
func addQuotes(value interface{}) bool {
    // don't add quotes if
   // value is null
    if value == "null" {return false}
    // value is a boolean
    if _, err := strconv.ParseBool(value); err == nil {return false}
    // value is a json object or array
    if strings.Contains(value, "\"") || strings.Contains(value, "[") || strings.Contains(value, "{") {return false}
    // value is a number
    if _, err := strconv.ParseFloat(value, 64); err == nil {return false}
    return true
}
```

then change this part to
```go
if addQuotes(p.Value) {
    return fmt.Sprintf(`{"op": "%s", "from": "%s", "path": "%s", "value": "%s"}`, p.Operation, p.From, p.Path, p.Value)
} else {
    return fmt.Sprintf(`{"op": "%s", "from": "%s", "path": "%s", "value": %s}`, p.Operation, p.From, p.Path, p.Value)
}
```
?

Reply:
thanks for the suggestion, will incorporate this.